SmoothStackProject1/src/utils/Validate.java

```diff
@@ -0,0 +1,25 @@
+package utils;
+
+import java.util.InputMismatchException;
+import java.util.regex.Pattern;
+
+public class Validate {
+	
+	public static boolean checkValidNum(String input) {
+		try {
+			Integer.parseInt(input);
+			return true;
+		}
+		catch(InputMismatchException e) {
+			return false;
+		}
+	}
+	
+	public static boolean checkValidPhoneNum(String input) {
```

Comment:
I do like the fact that you use regex to check if the phone number is in the right format

Reply:
Yes, if you're going to require phone numbers in a specific format, a regex is the right tool to use.